### ISSUE TITLE
Error using cargo on OSX

### ISSUE THREAD

**Issue by Phlosioneer** [rust-lang/cargo#1353]
Running the <code>cargo build</code> doesn't work; it gives the following error (when --verbose is used):
```rust
Process didn't exit successfully: `rustc - --crate-name - --crate-type dylib --crate-type bin --print=file-names` (status=101)
--- stderr
error: crate name cannot start with a hyphen: -
error: aborting due to previous error
```

Error was generated on a fresh project (made using <code>cargo new</code>)

Cargo and Rust version:
```
cargo 0.0.1-pre-nightly (43755c0 2015-02-19) (built 2015-02-21)
rustc 1.0.0-nightly (c4fe7d6ae 2015-02-23) (built 2015-02-24)
binary: rustc
commit-hash: c4fe7d6aea022330efb8d1b470cac030a859f8f5
commit-date: 2015-02-23
build-date: 2015-02-24
host: x86_64-apple-darwin
release: 1.0.0-nightly
```

Edit: OSX version 10.10.2

**Comment by erickt** [comment]
@Phlosioneer: this was fixed in #1344, and if you update to the 2015-02-25 nightly you shouldn't hit this error anymore.

**Comment by steveklabnik** [comment]
Yes, this is true.
